### ISSUE TITLE
Haven't received any confirmation mail from hacktoberfest even after having 4 accepted PR's .

### ISSUE THREAD

**Issue by kavyasree-2020** [vinitshahdeo/Hacktoberfest2021#565]
I haven't received any mail regarding completion of hacktoberfest contribution eventhough my 4 pr's got accepted (went into 14 days review period.

**Comment by SUKHJIT-CYBER** [comment]
Yes , PR is going under 14 days after that it will be fully completed

**Comment by kavyasree-2020** [comment]
Not really !!! Already I have 4pr's which went into 14 days review period.
Still didn't received any mail confirmation.

**Comment by kavyasree-2020** [comment]
You can have a look at it here 
![Screenshot_2021-10-23-08-22-28-318_com android chrome](https://user-images.githubusercontent.com/75531185/138540055-97cc0334-0e13-4b54-a78d-8f667682989e.png)

**Comment by hadviad0x15** [comment]
I guess 50000 count is completed that can be the case. Not sure though.

**Comment by kavyasree-2020** [comment]
May be , yeah !!!

**Comment by athifbinu** [comment]
what is the problem .
